diff -urN supertuxkart-0.9/lib/irrlicht/CMakeLists.txt supertuxkart-0.9-patched/lib/irrlicht/CMakeLists.txt
--- supertuxkart-0.9/lib/irrlicht/CMakeLists.txt	2015-04-21 21:32:20.020205746 +1000
+++ supertuxkart-0.9-patched/lib/irrlicht/CMakeLists.txt	2015-04-23 14:50:09.630910551 +1000
@@ -501,7 +501,7 @@
     set_source_files_properties(source/Irrlicht/MacOSX/OSXClipboard.mm PROPERTIES LANGUAGE C)
 endif()
 
-add_library(stkirrlicht ${IRRLICHT_SOURCES})
+add_library(stkirrlicht STATIC ${IRRLICHT_SOURCES})
 target_link_libraries(stkirrlicht ${PNG_LIBRARY} ${JPEG_LIBRARY} ${ZLIB_LIBRARY})
 
 
diff -urN supertuxkart-0.9/lib/wiiuse/CMakeLists.txt supertuxkart-0.9-patched/lib/wiiuse/CMakeLists.txt
--- supertuxkart-0.9/lib/wiiuse/CMakeLists.txt	2015-04-21 21:32:17.140191465 +1000
+++ supertuxkart-0.9-patched/lib/wiiuse/CMakeLists.txt	2015-04-23 14:50:40.116911938 +1000
@@ -38,12 +38,8 @@
     )
 endif()
 
-if(WIN32)
-    add_definitions("/DWIIUSE_STATIC")
-    add_library(wiiuse STATIC ${WIIUSE_SOURCES})
-else()
-    add_library(wiiuse ${WIIUSE_SOURCES})
-endif()
+add_definitions("/DWIIUSE_STATIC")
+add_library(wiiuse STATIC ${WIIUSE_SOURCES})
 
 if(MSVC)
     if(MSVC90)
